_includes/head.html

```diff
@@ -28,7 +28,11 @@
     <link href="//fonts.googleapis.com/css?family={{ site.theme_settings.google_fonts }}" rel="stylesheet" type="text/css">
     {% endif %}
     {% if page.path contains '_posts' %}
-     <link rel="amphtml" href="{{ page.id | prepend: '/amp' | prepend: site.baseurl_root | prepend: site.url }}">
+        {% if site.lang == 'fr' %}
+            <link rel="amphtml" href="{{ page.url | prepend: '/amp' | prepend: site.baseurl_root | prepend: site.url }}">
+        {% elsif site.lang == 'en' %}
```

Comment:
`elsif` ^^

Reply:
ca marche en plus :)

Reply:
https://stackoverflow.com/questions/12423658/how-do-i-chain-if-statements-in-jekyll

Reply:
je sais, mais ca m'accroche la rétine a chaque fois